### ISSUE TITLE
Ease issues with Import of MAME romsets

### ISSUE THREAD

**Issue by JON99999** [TeamShinkansen/Hakchi2-CE#250]
**Is your feature request related to a problem? Please describe.**
When I import multiple MAME romsets at once by drag and drop, two issues occur.  First, some files inside a zip are auto recognized as non-MAME roms and imported.  Second, every normal romset gets a dialog box asking if I want to import as a set.

**Describe the solution you'd like**
Provide a default setting for the import of "set" type files, or a flag on the first import that says to "treat remaining files as sets".

**Describe alternatives you've considered**
I have learned that, if I import as "file>Add More Games> As Is..." then the behavior I would like to happen happens.  All zips of MAME roms are brought in fully as a proper archives.  If there was a dialog box when a zip/7z with more than 1 item is drag and dropped, that let you turn everything on with the "As Is..." setting... Then that would solve it.

**Additional context**
As always, thanks for this wonderful tool!

**Comment by bslenul** [comment]
Yeah that "bug" is kinda annoying, I like the idea of having an option for that, something like a "Always import as-is when multiple files are detected inside the archive" (with a shorter name maybe :p).

But as a workaround, you can already add multiple roms at once using the "File > Add more games > As-Is..." option.
